src/file-uploader/file-uploader.component.ts

```diff
@@ -122,9 +145,16 @@ export class FileUploader {
 
 	@Output() filesChange = new EventEmitter<any>();
 
+	@Output() dropped = new EventEmitter<any>();
```

Comment:
Do we need a separate event for dropping files? Or can we just use `filesChange`?